lib/commands/session.js

```diff
@@ -13,6 +13,7 @@ commands.createSession = async function (caps) {
   this.validateDesiredCaps(caps);
   this.sessionId = UUID.create().hex;
   this.caps = caps;
+  this.rawCaps = _.cloneDeep(caps); // used for reset
```

Comment:
Thanks @Jonahss , looks like I'll need to search/replace `this.caps` by `this.opts` in a lot of places.